brokerapi/brokers/models/db.go

```diff
@@ -33,6 +34,14 @@ type ServiceBindingCredentials struct {
 	BindingId         string
 }
 
+func (sbc ServiceBindingCredentials) GetOtherDetails() map[string]string {
+	var creds map[string]string
+	if err := json.Unmarshal([]byte(sbc.OtherDetails), &creds); err != nil {
```

Comment:
any chance this one is empty?

Reply:
Can OtherDetails be anything? Or is is guaranteed to be creds?

Reply:
This should always contain stuff and should panic if it doesn't.